src/pages/_playground/api-explorer.tsx

```diff
@@ -293,6 +317,71 @@ export default function ApiExplorer() {
           />
         </div>
 
+        <div className="vads-u-margin-y--3">
+          <label className="vads-u-display--block vads-u-margin-bottom--1">
+            Filters
+          </label>
+          {queryState.filters.map((filter, index) => (
+            <div
+              key={index}
+              className="vads-u-display--flex vads-u-align-items--center vads-u-gap--1 vads-u-margin-bottom--2"
+            >
+              <input
+                type="text"
+                className="usa-input"
+                placeholder="Field name"
+                value={filter.field}
+                onChange={(e) => updateFilter(index, 'field', e.target.value)}
+                style={{ flex: 2 }}
+              />
+              <select
+                className="usa-select"
+                value={filter.operator}
+                onChange={(e) =>
+                  updateFilter(index, 'operator', e.target.value)
+                }
+                style={{ flex: 1 }}
+              >
+                <option value="=">=</option>
+                <option value="<>">≠</option>
+                <option value=">">&gt;</option>
+                <option value=">=">&gt;=</option>
+                <option value="<">&lt;</option>
+                <option value="<=">&lt;=</option>
+                <option value="IS NOT NULL">IS NOT NULL</option>
+                <option value="IS NULL">IS NULL</option>
```

Comment:
Fun fact: [This is also how you'd test for empty arrays](https://www.drupal.org/docs/core-modules-and-themes/core-modules/jsonapi-module/filtering#s-11-filtering-on-empty-array-fields). 🫠